### PR TITLE
Enable programmatic use of flipAxes(), address #50

### PR DESCRIPTION
**Pull request by joshhjacobson**
This approach to `flipAxisAndUpdatePCP` allows the function to be compatible with the double-click event based approach as well as the programmatic approach in the `flipAxes` side effect.

### CODE REVIEW

File: src/util/flipAxisAndUpdatePCP.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
     pc.flip(dimension);
     pc.brushReset(dimension);
-    select(this.parentElement)
+
+    // select(this.parentElement)
+    pc.selection
+      .select('svg')
+      .selectAll('g.axis')
+      .filter(function(d) {
+        if (d === dimension) {
```
Comment (BigFatDog) [inline src/util/flipAxisAndUpdatePCP.js]: .filter(d => d === dimension)

**Comment by BigFatDog** [comment]
@joshhjacobson 
Hi Josh, I've finished code review. I can see only one potential refactoring, nothing else.

**Comment by joshhjacobson** [comment]
Thanks for the suggestion, much cleaner!

**Comment by BigFatDog** [comment]
You're welcome. Thank YOU for this PR!

[v2.2.6](https://github.com/BigFatDog/parcoords-es/releases/tag/v2.2.6) has been released to deliver your fix.

**Pull request merged**